chef_master/source/custom_resources.rst

```diff
@@ -577,14 +577,14 @@ The ``converge_if_changed`` method may be used multiple times. The following exa
 
 .. code-block:: ruby
 
-   property :path, String, name_property: true
+   property :path, String
    property :content, String
    property :mode, String
 
-   load_current_value do
-     if ::File.exist?(new_resource.path)
```

Comment:
I think the simpler fix here is just to remove new_resource. I believe you can just access these as `path`

Reply:
Accessing them as path fails.  You get path = nil.  We either have to set one of the three attributes for the property or use a code block.

Reply:
I'm taking @MarkGibbons side on this; he's worked with it most recently.